### PR TITLE
Create CodeStyle.md

### PR DESCRIPTION
**Pull request by Klimonov**

### CODE REVIEW

**Review by kezlya — APPROVED** [review]
Perfect start! Thank you @Klimonov

File: CodeStyle.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+or
+<tag class="my-1 d-flex" />
+```
+
+### Use ::v-deep + scoped styles to customize custom vuetify components
+```js
+// ----------- bad
+<style>
+.class .v-input__icon--append .v-icon {
+    color: $red !important;
```
Comment (sadiolem) [inline CodeStyle.md]: maybe use css vars here as well (and in all other places in the file)

**Review by sadiolem — APPROVED** [review]
good job, LGTM

**Review by EgoriusRus — APPROVED** [review]
LGTM

**Pull request merged**